srcpkgs/ssss/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ssss'
+pkgname=ssss
+version=0.5
+revision=1
+build_style=gnu-makefile
+make_build_target="all"
+hostmakedepends="xmltoman"
+makedepends="gmp-devel"
+short_desc="Simple command-line implementation of Shamir's Secret Sharing Scheme"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://point-at-infinity.org/ssss/"
+distfiles="http://point-at-infinity.org/ssss/${pkgname}-${version}.tar.gz"
+checksum=5d165555105606b8b08383e697fc48cf849f51d775f1d9a74817f5709db0f995
+
+do_install() {
+	vbin ssss-combine
+	vbin ssss-split
+}
+
+post_install() {
```

Comment:
When a do_install() is necessary, you may omit the post_install() and move these files directly in the do_install.